packages/siop-oid4vp/lib/request-object/Payload.ts

```diff
@@ -1,4 +1,4 @@
-import { v4 as uuidv4 } from 'uuid'
+import { uuidv4 } from '@sphereon/oid4vci-common'
```

Comment:
This and the JWK thumbprint changes do result in the SIOP lib depending on the VCI lib. Right now the oid4vci-common lib still is very much vci only. Probably it would be wise to create a `common` lib, that vci-common and siop can depend upon, where we place the real common stuf